ketcher-autotests/tests/Reagents/SMILES-Format/reagents-smiles-format.spec.ts

```diff
@@ -97,20 +95,24 @@ test.describe('Reagents SMILES format', () => {
     },
   );
 
-  test(`Detection molecule as reagent
-  and write reagent information in "Extended SMILES" format in "Preview" tab`, async ({
-    page,
-  }) => {
-    /*
+  test.fixme(
```

Comment:
If you mark some test with `fixme`, then I propose to add a comment – may be it is not working at all, or it is very unstable or some other reason